models/solana_utils/solana_utils_token_accounts.sql

```diff
@@ -12,14 +15,18 @@
 
 WITH 
       distinct_accounts as (
-            --force
             SELECT
-                  token_mint_address
-                  , address 
-                  , max_by(token_balance_owner, block_time) as token_balance_owner --some account created before and then again after token owner schema change, so then it created dupes.
-                  , min(block_time) as created_at 
-            FROM {{ source('solana','account_activity') }}
-            WHERE token_mint_address is not null
+                  aa.token_mint_address
+                  , aa.address 
+                  , max_by(aa.token_balance_owner, aa.block_time) as token_balance_owner --some account created before and then again after token owner schema change, so then it created dupes.
+                  , min(aa.block_time) as created_at 
+            FROM {{ source('solana','account_activity') }} aa
+            LEFT JOIN {{ this }} old ON old.address = aa.address AND old.token_mint_address = aa.token_mint_address
```

Comment:
go ahead and remove this join, it adds extra noise when the merge can determine this for you automatically